python/hail/api1/context.py

```diff
@@ -467,8 +467,9 @@ def import_table(self, paths, key=[], min_partitions=None, impute=False, no_head
         information about possible field types. While this is a bit slower for large files, (the 
         file is parsed twice), the convenience is often worth this cost.
         
-        The ``delimiter`` parameter is a field separator regex. This regex follows the 
-         `Java regex standard <http://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html>`_.
+        The ``delimiter`` parameter is either a delimiter character (if a single character) or a field separator
+        regex (2 or more characters). This regex follows the `Java regex standard
```

Comment:
I personally don't like changing the interpretation on the length but I understand the argument for conciseness and python compatibility.  I'd probably argue for separate `delimiter_char` and `delimiter_regex` arguments.  No need to change anything, I just wanted the comment on the record.

Reply:
Yeah, but I don't totally like splitting it out into delim_char and delim_regex either.

Maybe something like `regex={True, False, 'auto'}` (auto is this behavior, and default)?

Reply:
Sigh, I don't really like that either, the interpreting delimiter is non-local.  I say we stick with your current change.